packages/core/src/generators/html.ts

```diff
@@ -40,6 +40,30 @@ type InternalToHtmlOptions = ToHtmlOptions & {
   namesMap: NumberRecord;
 };
 
+const attributeKeyExceptionsMap: { [key: string]: string } = {
+  class: 'className',
+};
+
+const updateKeyIfException = (key: string): string => {
+  return attributeKeyExceptionsMap[key] ?? key;
+};
+const normalizeAttributeKey = (key: string): string => {
+  return `"${key.replace(/\s/g, '')}"`;
```

Comment:
I'd drop the string interpolation since it's not needed:

```suggestion
  return key.replace(/\s/g, '');
```

Reply:
Just going over what we talked about. It turns out that the prettier plugin is the culprit so we don't need the replace anymore, however we do need the ""  somewhere so we removed the helper function and put it directly on the string itself.